aleph-client/src/multisig.rs

```diff
@@ -277,36 +261,35 @@ impl MultisigParty {
         Ok(multisig.when)
     }
 
-    /// Checks whether `member_idx` is a proper position for `self.members`.
-    fn ensure_index(&self, member_idx: usize) -> Result<()> {
-        ensure!(
-            member_idx < self.members.len(),
-            MultisigError::IncorrectMemberIndex
-        );
-        Ok(())
+    /// Checks whether `connection` is signed by some member and if so, returns their index.
+    fn map_signer_to_member(&self, connection: &SignedConnection) -> Result<usize> {
```

Comment:
nit: map_signer_to_index or return the accountId

Reply:
renamed